src/utils/db/schema/lyricTable.ts

```
@@ -0,0 +1,8 @@
+import { int, sqliteTable, text } from 'drizzle-orm/sqlite-core';
```

ESLint

[error] Resolve error: Cannot find module 'babel-plugin-module-resolver'
Require stack:
- /home/jailuser/git/node_modules/eslint-import-resolver-babel-module/lib/index.js
- /home/jailuser/git/node_modules/eslint-module-utils/resolve.js
- /home/jailuser/git/node_modules/eslint-plugin-import/lib/rules/no-unresolved.js
- /home/jailuser/git/node_modules/eslint-plugin-import/lib/index.js
- /home/jailuser/git/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1248:15)
    at Module._load (node:internal/modules/cjs/loader:1074:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Module.require (node:internal/modules/cjs/loader:1339:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (/home/jailuser/git/node_modules/eslint-import-resolver-babel-module/lib/index.js:8:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)

(import/namespace)